game/fighter.py

```diff
@@ -84,15 +93,17 @@ def resource_path(relative_path):
             base_path = os.path.abspath(".")
 
         return os.path.join(base_path, relative_path)
-
+    
     def move(self, screen_width, screen_height, surface, target, obstacles):
-        GRAVITY = 1
+        GRAVITY = 2
```

Comment:
players cant jump as high now and cant jump to high grounds anymore with gravity = 2